sync/class.jetpack-sync-actions.php

```diff
@@ -227,61 +227,43 @@ static function jetpack_cron_schedule( $schedules ) {
 		return $schedules;
 	}
 
-	// try to send actions until we run out of things to send,
-	// or have to wait more than 15s before sending again,
-	// or we hit a lock or some other sending issue
 	static function do_cron_sync() {
-		if ( ! self::sync_allowed() ) {
-			return;
-		}
-
-		self::initialize_sender();
-
-		$time_limit = Jetpack_Sync_Settings::get_setting( 'cron_sync_time_limit' );
-		$start_time = time();
-
-		do {
-			$next_sync_time = self::$sender->get_next_sync_time( 'sync' );
-
-			if ( $next_sync_time ) {
-				$delay = $next_sync_time - time() + 1;
-				if ( $delay > 15 ) {
-					break;
-				} elseif ( $delay > 0 ) {
-					sleep( $delay );
-				}
-			}
-
-			$result = self::$sender->do_sync();
-		} while ( $result && ( $start_time + $time_limit ) > time() );
+        self::do_cron_sync_by_type('sync');
```

Comment:
Tiny detail: could you add spaces in the braces, as per [the WordPress coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage)? To make this easier, I'd recommend installing Code Sniffer rules:
https://github.com/Automattic/jetpack/blob/master/docs/coding-guidelines.md#general